packages/core/src/sass/app/modules/account-signup.scss

```diff
@@ -43,12 +43,16 @@
         transform: translate(-50%);
 
         @include mobile {
-            width: calc(95vw - 20px);
-            height: 36.5rem !important;
+            width: calc(90vw - 10px);
             padding: 6rem 4rem;
             background-color: var(--general-main-2);
         }
     }
+    @include mobile {
+        &__password-selection {
+            height: 36.5rem !important;
```

Comment:
We shouldn't use `!important` in our styles. is there any way to remove that?

Reply:
it existed in the code. I just moved it to other line

Reply:
Alrighty